docs/content/ChoiceGroup.mdx

```diff
@@ -0,0 +1,382 @@
+---
+title: ChoiceGroup
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/ChoiceGroup/ChoiceGroup.tsx
+storybook: '/react/storybook/?path=/story/forms-choicefieldset--radio-group'
+---
+
+import {ChoiceGroup, Box} from '@primer/components'
+import {CheckIcon, XIcon, AlertIcon} from '@primer/octicons-react'
+import {ComponentChecklist} from '../src/component-checklist'
+
+A `ChoiceGroup` is used to render a related set of checkbox or radio inputs.
+
+## Examples
+
+### Basic
+
+```jsx live
+<ChoiceGroup>
```

Comment:
Shall we update the `FormControl` [radio and checkbox example here](https://primer-react-h5y9n1lso-primer.vercel.app/react/ChoiceGroup) to use this component? These examples feel very similar now. Maybe adding a Flash or internal link pointing to `ChoiceGroup` so users know it exists.

Reply:
Yes, definitely! Good catch.